### PR TITLE
Quality of Life Changes

### PR DESCRIPTION
**Pull request by Boomatang**
Added package-lock.json to .gitignore to aid scanning

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,6 +30,7 @@ hs_err_pid*
 node_modules/
 npm-debug.log
 yarn-error.log
+package-lock.json
```
Comment (ziccardi) [inline .gitignore:33]: According to [npm docs](https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md) package-lock.json is meant to be committed

**Review by ziccardi — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (secondsun) [reply .gitignore:33]: @ziccardi this is the react native client and package-lock actually breaks it.

**Review by secondsun — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -34,4 +34,4 @@
     "eslint": "^6.8.0",
     "@react-native-community/eslint-config": "^1.1.0"
   }
-}
\ No newline at end of file
+}
```
Comment (ziccardi) [inline package.json:37]: Please remove this newline: it gets automatically removed by the version bump workflow.

**Review by ziccardi — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Boomatang) [reply package.json]: I was afraid you you say that :) newline has been removed removed

**Review by ziccardi — APPROVED** [review]

**Pull request merged**